cvat-ui/src/components/tasks-page/empty-list.tsx

```diff
@@ -31,6 +31,10 @@ export default function EmptyListComponent(): JSX.Element {
             <Row type='flex' justify='center' align='middle'>
                 <Col>
                     <Link to='/tasks/create'>create a new task</Link>
+                    &nbsp;
```

Comment:
Let's remove it ```&nbsp;```
If you need the text on different lines, try to wrap the lines in <div> for example, or use <br />